facebook_bot.js

```diff
@@ -99,10 +100,18 @@ if(ops.lt === false && ops.ltsubdomain !== null) {
     process.exit();
 }
 
+var access_token = process.env.page_token;
```

Comment:
this was inherited from the previous sample code. I can certainly update it to keep the consistency.